site/src/pages/components/accordion.explainer.mdx

```diff
@@ -151,6 +154,21 @@ violate the exclusivity semantics?
 (This applies both to insertion by the parser
 and insertion through other methods like setting `innerHTML`.)
 
+**Proposed solution**: Enforce exclusivity only for
+user interactions,
+use of the [`open` IDL attribute](https://html.spec.whatwg.org/multipage/C#dom-details-open),
+or setting the `open` attribute via `setAttribute`
+(or `setAttributeNS`, `setAttributeNode`, or `setAttributeNodeNS`).
+Do not enforce exclusivity when the `open` attribute is set by the parser,
+when it is set as a result of cloning a node,
+or when an element is moved to a new document.
```

Comment:
nit: maybe include "*removal* from a document", not just moving to a new document? I.e. there's a problem at both ends of that move.

Reply:
I don't think there's a problem with removal since this notion of exclusivity accepts either 0 or 1 elements being open.  (@bkardell pointed out that there is some usage of exactly-1 exclusivity, which should perhaps be discussed somewhere!)